pkg/provisioner/ironic/ironic.go

```diff
@@ -931,6 +931,12 @@ func (p *ironicProvisioner) Deprovision(deleteIt bool) (result provisioner.Resul
 			nodes.ProvisionStateOpts{Target: nodes.TargetManage},
 		)
 
+	case nodes.Inpsecting, nodes.InspectWait:
```

Comment:
typo: `Inspecting`?

Reply:
The typo is in the gophercloud code. When we update to a new version, the const name will be fixed and we can update this spot, too.